src/struct_mv/protos.h

```diff
@@ -72,6 +72,14 @@ HYPRE_Int hypre_AddIndexes ( hypre_Index index1, hypre_Index index2, HYPRE_Int n
 HYPRE_Int hypre_SubtractIndexes ( hypre_Index index1, hypre_Index index2, HYPRE_Int ndim,
                                   hypre_Index result );
 HYPRE_Int hypre_IndexesEqual ( hypre_Index index1, hypre_Index index2, HYPRE_Int ndim );
+HYPRE_Int
+hypre_IndexesGreater( hypre_Index  index1,
```

Comment:
Very minor comment: I was wondering if we should have the return data type (`HYPRE_Int`) in the same line as the function name as we have in the other prototypes

Reply:
Actually, I was wondering if we should make our lives easier and just use copy-paste from now on in protos.h. :)  The reason they were one-liners before is historical - we used to have a script that automatically generated the prototypes from the C-files.  Now that we are doing it by hand (which we've been doing for a while, of course), it probably makes more sense to make it simple for developers.  These are internal to hypre, so the format choice we make only affects developers.

Reply:
That makes sense! I guess bringing back the script would be even better :)